src/routes/+page.svelte

```diff
@@ -28,6 +30,13 @@
 
   core.initialize()
 
+  // Initialize Webnative
+  async function init() {
+    await initialize()
+  }
+
+  init().then().catch(console.log)
```

Comment:
nitpick: i'm not the biggest fan of mixing `then`/`catch` syntax with `async`/`await`. Would it be possible to add a `try`/`catch` to the `init` function on line 34 instead? also, not necessary because this is just a personal preference of mine 😅

Reply:
The `init()` call on its own has a floating promise TypeScript warning. Tried adding a `try`/`catch` in the body of the function, but we still have that warning. Strangely, I don't see that warning where we are doing a similar thing in WAT. 🤔 

I dropped then `then` which wasn't really doing anything and will merge it like that for now. Totally open to changing it later if we can sort out the floating promise warning.

Reply:
ahh very strange 🤔 I'd assume

```
  async function init() {
    try {  
      await initialize()
    } catch (error) {
      console.error(error)
    }
  }
```
would just work. Oh well haha, we can revisit it 👍🏼